### DISCUSSION TITLE
Limit of selecting files or any m2m relation

### DISCUSSION THREAD

**Discussion by Dominic-Preap** [directus/directus#19556] [Feature Requests]
### Summary

I'm not sure if this already exist. I would like to have a limit file select to max 3 files but I'm not sure how to achieved it. 
![image](https://github.com/directus/directus/assets/14802170/4cf85af9-de9d-4821-96fb-da49ceff2e25)

Also checking with `validation` but does not have anything for example greater than `your-number`.

![image](https://github.com/directus/directus/assets/14802170/c0c73665-a7ab-4194-a013-035a574b3248)

If this can already be achieved, please let me know, I will close this request. 

### Motivation

It would be nice to have more control the max selecting of any m2m relationship data not just only files. And the number can  be configurable either by either load from `.env` or other related custom fields or just static number.  

### Detailed Design

We can add some filter rules in validation check like Less than, Greater than

### Requirements List

None

### Drawbacks

None

### Alternatives

None

### Adoption Strategy

None

**Comment by Dominic-Preap** [comment]
@rijkvanzanten could you give me a bit of insight?

**Reply by DanielBiegler** [reply]
Hi there, I like this suggestion but in the future please refrain from tagging team members directly for bumps/updates. Thank you!

**Reply by Dominic-Preap** [reply]
Sorry about that, will look forward in the future. Thank you

**Reply by DanielBiegler** [reply]
Regarding questions where you want to figure out how something works, our [Discord](https://directus.chat/) is very helpful.

Regarding this feature request, in my opinion this is a nice addition. Adding other comparisons like a specific length could be useful too.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
